pkg/sql/logictest/testdata/logic_test/crdb_internal_tenant

```diff
@@ -11,6 +11,8 @@ SELECT node_id, name FROM crdb_internal.leases ORDER BY name
 0  eventlog
 0  jobs
 0  locations
+0  protected_ts_meta
```

Comment:
BTW, what's this about?

Reply:
I'm not a 100% sure, but a little digging led me to https://github.com/cockroachdb/cockroach/pull/69008 where we switched to leasing all system descriptors except a few in a deny list. The PTS tables are not in this deny list so I guess we expect to see them here?